src/test/java/org/jenkinsci/plugins/durabletask/BourneShellScriptTest.java

```diff
@@ -480,21 +482,37 @@ private Slave prepareDockerPlatforms() throws Exception {
     @Test public void binaryCaching() throws Exception {
         assumeTrue(!Objects.equals(System.getenv().get("SKIP_DURABLE_TASK_BINARY_GENERATION"), "true") && !platform.equals(TestPlatform.UBUNTU_NO_BINARY));
```

Comment:
Should we get rid of the stuff related to the environment variable now? Is there any other code that uses it?

Reply:
```suggestion
        assumeFalse(platform.equals(TestPlatform.UBUNTU_NO_BINARY));
```

Reply:
fixed in 388a7bf1c9a1ba7c8e00f1b367c01daaa95b1c0a